src/platforms/virtual/graphics/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+include_directories(
+    ${server_common_include_dirs}
+)
+
+add_library(
+    mirplatformgraphicsvirtualobjects OBJECT
+
+    platform.cpp
+    display.cpp
+    display_configuration.cpp
+)
+
+add_library(
+    mirplatformgraphicsvirtualobjects-symbols OBJECT
+
+    graphics.cpp
+)
+
+target_link_libraries(mirplatformgraphicsvirtualobjects-symbols
+    PUBLIC
+    mirplatform
+    mircommon
+    mircore
+)
+
+target_link_libraries(
+    mirplatformgraphicsvirtualobjects
+
+    PRIVATE
+    mirplatform
+)
```

Comment:
I would expect the virtual platform to be entirely in the `virtual` subdirectory, instead the source files are in the `graphics` directory, as is a (possibly redundant) `mirplatformgraphicsvirtualobjects` OBJECT library. Only the .so build is in the `virtual` subdirectory

Reply:
@mattkae this file looks rogue now?